pcf/cli/commands/apply.py

```diff
@@ -0,0 +1,34 @@
+""" Logic for pcf apply command """
+
+import click
+from pcf.cli.utils import execute_applying_command, click_options
+from pcf.cli.commands import COMMON_APPLY_OPTIONS
+
+
+@click.command(name="apply", short_help="Set a desired state and apply")
+@click.option(
+    "-s",
+    "--state",
+    type=click.Choice(["running", "stopped", "terminated"], case_sensitive=False),
+    default="running",
```

Comment:
we might want to force them to specify desired state instead of defaulting to running